src/bs_size.c

```diff
@@ -134,7 +134,7 @@ static char *replace_char_with_str (const char *str, char orig, const char *new)
         return NULL;
 
     next = str;
-    for (next=strchr (next, orig); next; count++, next++);
+    for (next=strchr (next, orig); next; count++, next=strchr (++next, orig));
```

Comment:
Ooops, good catch, thanks! I wish there was some sane way to test ``static`` functions in *C* plus a way to check something like this.